src/cli/utils/restore_config.go

```diff
@@ -0,0 +1,61 @@
+package utils
+
+import (
+	"context"
+
+	"github.com/alcionai/clues"
+	"github.com/spf13/cobra"
+
+	"github.com/alcionai/corso/src/cli/flags"
+	. "github.com/alcionai/corso/src/cli/print"
+	"github.com/alcionai/corso/src/internal/common/dttm"
+	"github.com/alcionai/corso/src/pkg/control"
+)
+
+type RestoreCfgOpts struct {
+	Collisions  string
+	Destination string
+
+	Populated flags.PopulatedFlags
+}
+
+func makeRestoreCfgOpts(cmd *cobra.Command) RestoreCfgOpts {
+	return RestoreCfgOpts{
+		Collisions:  flags.CollisionsFV,
+		Destination: flags.DestinationFV,
+
+		Populated: flags.GetPopulatedFlags(cmd),
+	}
+}
+
+// validateRestoreConfigFlags checks common flags for correctness and interdependencies
+func validateRestoreConfigFlags(opts RestoreCfgOpts) error {
+	_, populated := opts.Populated[flags.CollisionsFN]
+	_, valid := control.ValidCollisionPolicies()[control.CollisionPolicy(flags.CollisionsFV)]
+
+	if populated && valid {
```

Comment:
Seems like this needs adjusted. I'm taking a guess here
```suggestion
	if !populated || !valid {
```

Reply:
bargle.  `if populated && !valid {`